spyder/utils/programs.py

```diff
@@ -798,6 +798,29 @@ def run_terminal_thread():
         raise NotImplementedError
 
 
+def check_version_range(module_version, version):
```

Comment:
```suggestion
def check_version_range(module_version, version_range):
```

I think it's clearer if we rename here `version` by `version_range`.